mkfs/mkfs.c

```diff
@@ -383,7 +383,7 @@ static int exfat_create_root_dir(struct exfat_blk_dev *bd,
 static void usage(void)
 {
 	fprintf(stderr, "Usage: mkfs.exfat\n");
-	fprintf(stderr, "\t-L string | --volume-label=string    Set volume label\n");
+	fprintf(stderr, "\t-L | --volume-label                  Set volume label\n");
```

Comment:
I think the options with an argument such as "--volume-label=string" look better. I am curious about your opinion.

Reply:
Okay. Need to update other options in help messagea also.